### PR TITLE
Adding "enstat" and "openscienceplot_matplotlib"

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tdegeus** [comment]
Ready for review @conda-forge/help-python

File: recipes/enstat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python >=3
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3
+    - numpy
+
+test:
+  imports:
+    - enstat
```
Comment (chrisburr) [inline recipes/enstat/meta.yaml:29]: Is there any reason for not including `pip check` here?
```suggestion
    - enstat
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/openscienceplot_matplotlib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - setuptools_scm
+  run:
+    - python >=3
+    - numpy
+    - matplotlib
+    - h5py
+
+test:
+  imports:
+    - openscienceplot_matplotlib
```
Comment (chrisburr) [inline recipes/openscienceplot_matplotlib/meta.yaml:31]: Is there any reason for not including `pip check` here?
```suggestion
    - openscienceplot_matplotlib
  commands:
    - pip check
  requires:
    - pip
```

Comment (tdegeus) [reply recipes/openscienceplot_matplotlib/meta.yaml:31]: That I didn't know it existed! Thanks!

**Pull request merged**